diff --git a/Android.bp b/Android.bp
index be44509..ccac5a9 100644
--- a/Android.bp
+++ b/Android.bp
@@ -36,11 +36,16 @@ rust_library {
         "libbyteorder",
         "libcrc32fast",
         "libflate2",
     ],
     apex_available: [
         "//apex_available:platform",
         "com.android.virt",
     ],
     product_available: true,
     vendor_available: true,
+    arch: {
+        arm: {
+            rustlibs: ["libcrossbeam_utils"],
+        },
+    },
 }
